packages/plugin/webpack/src/WebpackConfig.ts

```diff
@@ -164,32 +164,17 @@ export default class WebpackConfigGenerator {
     );
   }
 
-  async getRendererConfig(entryPoints: WebpackPluginEntryPoint[]): Promise<Configuration> {
+  async getRendererConfig(entryPoints: WebpackPluginEntryPoint[]): Promise<Configuration[]> {
     const rendererConfig = this.resolveConfig(this.pluginConfig.renderer.config);
-    const entry: webpack.Entry = {};
-    for (const entryPoint of entryPoints) {
-      const prefixedEntries = entryPoint.prefixedEntries || [];
-      entry[entryPoint.name] = prefixedEntries.concat([entryPoint.js]);
-    }
-
     const defines = this.getDefines(false);
-    const plugins = entryPoints
-      .filter((entryPoint) => Boolean(entryPoint.html))
-      .map(
-        (entryPoint) =>
-          new HtmlWebpackPlugin({
-            title: entryPoint.name,
-            template: entryPoint.html,
-            filename: `${entryPoint.name}/index.html`,
-            chunks: [entryPoint.name].concat(entryPoint.additionalChunks || []),
-          }) as WebpackPluginInstance
-      )
-      .concat([new webpack.DefinePlugin(defines), new AssetRelocatorPatch(this.isProd, !!this.pluginConfig.renderer.nodeIntegration)]);
-    return webpackMerge(
-      {
-        entry,
+
+    return entryPoints.map((entryPoint) => {
+      const config = webpackMerge(rendererConfig || {}, {
```

Comment:
On further inverstigation this was an issue in my test project because my webpack config had `target: 'electron-renderer` set, which is not the documented way to use this plugin. After removing that it works just as I expect.